src/pages/wishlist/Wishlist.jsx

```diff
@@ -0,0 +1,14 @@
+import { useProductContext } from "../../contexts/products"
+import { WishListItemCard } from './../../components/Card/WishListItemCard';
+
+export const WishList = () => {
+    const { state: { itemsInWishList } } = useProductContext();
+    return (
+        <div class="align-text-center">
+            <span>Number of items in wishlist {itemsInWishList.length}</span>
+            {itemsInWishList.map((product) => {
+                return <WishListItemCard class="product" key={product._id} product={product} />
+            })}
+        </div>
```

Comment:
suggestion: You can also show some message when there are zero items in Wishlist. your wishlist is empty, continue shopping.